code/modules/client/preferences.dm

```diff
@@ -403,6 +403,10 @@ var/global/list/time_prefs_fixed = list()
 
 	character.backpack_setup = new(backpack, backpack_metadata["[backpack]"])
 
+	if(length(traits))
+		for(var/trait_type in traits)
+			character.set_trait(trait_type, (traits[trait_type] || TRAIT_LEVEL_EXISTS))
```

Comment:
nothing that this is probably fine as limbs/organs have been created by this point. i was initially a little worried about amputation traits etc but it seems a-okay